From 8c3491c2e9135d853d0c7ca4053374b9b624d845 Mon Sep 17 00:00:00 2001
From: Lubomir Rintel <lkundrak@v3.sk>
Date: Wed, 23 Apr 2014 11:31:22 +0200
Subject: [PATCH] Remove unnecessary test dependencies

ccm, pytz - Only used by integration tests
unittest2 - Part of python
PyYAML - Not used at all
---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index 11f7322..ecf9e0c 100644
--- a/setup.py
+++ b/setup.py
@@ -196,7 +196,7 @@ def run_setup(extensions):
         packages=['cassandra', 'cassandra.io'],
         include_package_data=True,
         install_requires=dependencies,
-        tests_require=['nose', 'mock', 'ccm', 'unittest2', 'PyYAML', 'pytz'],
+        tests_require=['nose', 'mock'],
         classifiers=[
             'Development Status :: 5 - Production/Stable',
             'Intended Audience :: Developers',
-- 
1.9.0

